Source/SwiftLintFramework/Rules/ProhibitedSuperRule.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ProhibitedSuperRule.swift
+//  SwiftLint
+//
+//  Created by Aaron McTavish on 12/12/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct ProhibitedSuperRule: ConfigurationProviderRule, ASTRule, OptInRule {
```

Comment:
Yeah actually, I wrote this before realizing that this rule didn't check for superclass inheritance to make sure that only the correct methods were being flagged, so opt-in is probably best. 👌